javav2/example_code/appsync/src/main/java/com/example/appsync/DeleteDataSource.java

```diff
@@ -0,0 +1,72 @@
+// snippet-sourcedescription:[DeleteDataSource.java demonstrates how to delete an AWS AppSync data source.]
+//snippet-keyword:[AWS SDK for Java v2]
+// snippet-service:[AWS AppSync]
+// snippet-keyword:[Code Sample]
+// snippet-sourcetype:[full-example]
+// snippet-sourcedate:[04-01-2022]
+// snippet-sourceauthor:[scmacdon - AWS]
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+package com.example.appsync;
+
+//snippet-start:[appsync.java2.del_ds.import]
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.appsync.AppSyncClient;
+import software.amazon.awssdk.services.appsync.model.AppSyncException;
+import software.amazon.awssdk.services.appsync.model.DeleteDataSourceRequest;
+//snippet-end:[appsync.java2.del_ds.import]
+
+/**
+ * Before running this Java V2 code example, set up your development environment, including your credentials.
+ *
+ * For more information, see the following documentation topic:
+ *
+ * https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/get-started.html
+ */
+public class DeleteDataSource {
+
+    public static void main(String[] args) {
+
+        final String usage = "\n" +
+                "Usage: " +
+                "   <apiId> <dsName> \n\n" +
+                "Where:\n" +
+                "   apiId - the id of the API (You can obtain the value from the AWS Management console). \n\n" +

```

Comment:
(You can obtain the value from the AWS Management console). --> (You can get this value from the AWS Management Console).

Reply:
fixed